### ISSUE TITLE
WAVE reporting errors on Likert component

### ISSUE THREAD

**Issue by Magnusrm** [Altinn/app-frontend-react#1037]
### Description of the bug

NOTE: The accessibility for screen reader (VoiceOver) is working as intended using aria-labels. This is an issue noticed by the accessibility testing tool WAVE.

The testing tool Wave is giving a warning that the component contain labels, but that they do not have any content. It has been tested with a screen reader and every bit of information in relation to the radio button was read. (Tested with VoiceOver) Maybe look into if the RadioButtons should be structured in another way?

<img width="297" alt="image" src="https://user-images.githubusercontent.com/32294735/227173020-ad07f65d-0a44-4ecd-a725-0227dd12f545.png">
<img width="491" alt="image" src="https://user-images.githubusercontent.com/32294735/227181654-fb7f8106-f642-4a2c-a568-f6883166f93f.png">

### Steps To Reproduce

1. Go to an app with a likert component
2. Run wave or another testing tool

**Comment by lasseklovstad** [comment]
Vi har merket med NVDA at den leser opp spørsmål/svar dobbelt så mange ganger når man tabber seg gjennom Likert-skala'en. Dette tror jeg er fordi den først skal lese opp posisjonen i tabellen og deretter radioknappen. 

Idag ser strukturen til likerten slik ut:
```html
<table>
   <thead>
      <tr>
         <th id="header-1">Spørsmål</th>
         <th id="header-2">Mindre</th>
         <th id="header-3">Mye</th>
         <th id="header-4">Mest</th>
      </tr>
   </thead>
   <tbody>
      <tr aria-labelled-by="row-1-question">
         <td id="row-1-question">Hvor mye trives du?</td>
         <td><input type="radio" aria-labelled-by="row-1-question header-2" /></td>
         <td><input type="radio" aria-labelled-by="row-1-question header-3" /></td>
         <td><input type="radio" aria-labelled-by="row-1-question header-4" /></td>
      </tr>
   </tbody>
</table>
```

Jeg har undersøkt microsft forms og de overskriver role på hver rad slik:
```html
<table>
   <thead>
      <tr>
         <th id="header-1">Spørsmål</th>
         <th id="header-2">Mindre</th>
         <th id="header-3">Mye</th>
         <th id="header-4">Mest</th>
      </tr>
   </thead>
   <tbody>
      <tr role="radiogroup" aria-labelled-by="row-1-question">
         <td id="row-1-question">Hvor mye trives du?</td>
         <td><input type="radio" aria-labelled-by="row-1-question header-2" /></td>
         <td><input type="radio" aria-labelled-by="row-1-question header-3" /></td>
         <td><input type="radio" aria-labelled-by="row-1-question header-4" /></td>
      </tr>
   </tbody>
</table>
```
Ved å gjøre dette fungerer hver rad som en normal radiogruppe istedenfor en tabell. Det eneste som er litt rart for skjermleser brukeren er at man får lest opp at det er en tabell med en rad (header). Og man tror kanskje det er noe feil med radene... Litt usikker her.
